depends/README_USAGE.md

```diff
@@ -0,0 +1,38 @@
+To build dependencies for the current arch+OS:
+
+    make
+
+To build for another arch/OS:
+
+	make HOST=host-platform-triplet && make HOST=host-platform-triplet
```

Comment:
whoops. This is just busted c/p from an initial version of depends that worked differently (required make && make install). Wrong on several levels :)

The second part can just be dropped.